src/solc_0.8/polygon/root/asset/AssetERC721Tunnel.sol

```diff
@@ -29,6 +29,7 @@ contract AssetERC721Tunnel is FxBaseRootTunnel, IERC721MandatoryTokenReceiver, E
         address _trustedForwarder,
         uint256 _maxTransferLimit
     ) FxBaseRootTunnel(_checkpointManager, _fxRoot) {
+        require(_maxTransferLimit > 0, "PolygonAssetERC721Tunnel: _maxTransferLimit invalid");
```

Comment:
PolygonAssetERC721Tunnel => AssetERC721Tunnel

Reply:
I have made the required change.